src/query/storage/interface.go

```diff
@@ -25,6 +25,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/m3db/m3/src/dbnode/encoding"
```

Comment:
we shouldn't have dbnode specific stuff in our storage interface. Eventually, I was planning to move out local storage into stores/m3db as well.